getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -77,7 +77,7 @@ To access the GitHub security insights widget on the Overview page in {product-s
 --
 
 . Navigate to the *Helm* tab and select *Upgrade*.
-. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from Secrets*, add `rhdh-secrets` as value.
+. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from existing Secrets*, add `rhdh-secrets` as value.
```

Comment:
"... from secrets" was correct

<img width="813" alt="Screenshot 2024-01-25 at 5 25 42 PM" src="https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/22490998/9c97b3ba-da2f-46a0-a0fa-fca5ba7fc398">

Reply:
The screenshot in Google doc is outdated. The updated screen shows "existing" in the field name: 
![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/59050394/69043ca0-4f1d-489a-a558-69654d3cfc0f)